test/morph_test.tsx

```diff
@@ -51,16 +52,16 @@ test("morph text input value undefined", () => {
 
 test("morph option selected", async () => {
   const select = populate(
-    <select>
-      <optgroup>
-        <option>Foo</option>
-      </optgroup>
-      <option selected>Bar</option>
-      <option>Baz</option>
-    </select>,
-  ) as HTMLInputElement
+      <select>
```

Comment:
Seems like a change from my local prettier settings, I am happy to revert if desired.